b3lb/rest/b3lb/lb.py

```diff
@@ -130,18 +147,19 @@ def get_node_params_by_lowest_workload(cluster_group):
         return None
 
 
-def get_slide_body_for_post(slide):
-    slide_path = "rest/slides/{}".format(slide)
-    try:
-        slide_file = open(slide_path, "rb")
-        body = '<modules><module name="presentation"><document name="{}">{}</document></module></modules>'.format(slide, base64.b64encode(slide_file.read()).decode())
-        slide_file.close()
-    except FileNotFoundError:
-        body = None
+def get_slide_body_for_post(asset, secret):
+    if asset.slide_filename:
```

Comment:
I would prefer to have a @property for the current filename. The filename should consist of two parts:

'.'.join(`slide_filename` with default=`tenant.slug.lower()`, mimetype.split('/')[-1])

**TBD**

Reply:
removed secret because of redundancy